blog/content/post/2021-04-21-right_to_repair.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Right to Repair - and why is it important?"
+abstract: "Let's observe the struggles of today's repairman,
+          why a right to repair is important and how 3mdeb can help."
+cover: /covers/right_to_repair.png
```

Comment:
@mikebdp2 what are the redistribution rights for picture?

Reply:
@pietrushnic Redistribution rights are unknown, and all the websites are using it without mentioning an author. Guess, a public domain in practice - just like the already-merged [SW.png](https://blog.3mdeb.com/covers/SW.png) which I wanted to use by myself but noticed it's already taken.